esp/templates/survey/review.html

```diff
@@ -12,19 +12,68 @@
 <style type="text/css">
 .nocheckmark { border: 1px solid black; }
 </style>
-<br /><br />
-<h1>Survey Responses for {{ program.niceName }} </h1>
+<a name="top"></a><h1>Survey Responses for {{ program.niceName }} </h1>
+
+{% if teacher_form %}
```

Comment:
I'm morphed into a non-admin teacher and I still see the search form.

Reply:
(also tried just logging into this non-admin teacher and same thing so don't blame the morphing :P)

Reply:
Yeah, I noticed this when I was testing. I'm honestly not sure what's going on. I'll look into it again.

Reply:
I believe this should now be fixed. Let me know if you're still having issues.